File: sdk-py/examples/deploy.py
import boxty as bx

app = bx.App("my-ml-service")

@app.function(image="pytorch:latest", gpu="A100", cpu=8, memory=32768)
@bx.web_endpoint(method="POST")
def predict(data: dict):
    return {"prediction": "ok"}

app.deploy()

File: sdk-js/examples/deploy.js
import { Client } from 'boxty';

const client = new Client({ apiKey: 'bxty_...' });
const app = await client.createApp('ws-1', 'env-1', 'my-ml-service', 'pytorch:latest', 8, 32768, 'A100');
console.log('Deployed', app.id);
